### PR TITLE
Fix Staking API Max Nominations

### LINKED ISSUE #5492: consts.staking.maxNominations is not fetching the correct value
Screenshot 2024-01-10 052400

**Affected Pages**
https://wiki.polkadot.network/docs/learn-staking
https://wiki.polkadot.network/docs/learn-nominator

Note : Search for the keyword " [object Object] " the issue is affecting many pages.

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/general/kusama/kusama-parameters.md [hunk 1]
```diff
@@ -71,8 +71,8 @@ The Kusama Council consists of up to 19 members and up to 19 runners up.
 ### Staking, Validating, and Nominating
 
 A maximum of
-{{ polkadot: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :polkadot }}
-{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
+{{ polkadot: <RPC network="polkadot" path="staking.nominationsquota" defaultValue={16}/> :polkadot }}
```
Comment (profradha) [inline docs/general/kusama/kusama-parameters.md]: I think Keith advised using both polkadot and kusama conditional renders even if we are just talking about one network. This page is kusama parameters, so in both Polkadot Wiki and Kusama Guide, this should just show kusama values.

```suggestion
{{ polkadot: <RPC network="kusama" path="staking.nominationsquota" defaultValue={24}/> :polkadot }}
```

File: docs/general/kusama/kusama-parameters.md [hunk 2]
```diff
@@ -71,8 +71,8 @@ The Kusama Council consists of up to 19 members and up to 19 runners up.
...
+{{ kusama: <RPC network="kusama" path="staking.nominationsquota" defaultValue={24}/> :kusama }}
 validators can be nominated by a nominator on Kusama.
```
Comment (profradha) [inline docs/general/kusama/kusama-parameters.md]: You can see that this number is for Kusama here.

**Review by DrW3RK — COMMENTED** [review]
This fix is not fetching the value from runtime. Instead, it is always displaying the default value provided. You can test it out by modifying the default values to say 8, an it will always show 8.

We may have to implement a separate component for this, as the runtime call expects balance as input. I'm okay to approve this patch, as it takes care of the issue temporarily with the default values.

File: docs/general/kusama/kusama-parameters.md [hunk 3]
```diff
@@ -70,10 +70,8 @@ The Kusama Council consists of up to 19 members and up to 19 runners up.
 
 ### Staking, Validating, and Nominating
 
-A maximum of
-{{ polkadot: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :polkadot }}
-{{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
-validators can be nominated by a nominator on Kusama.
+A maximum of {{ polkadot: 24 :polkadot }} {{ kusama: 24 :kusama }} validators can be nominated by a
```
Comment (profradha) [inline docs/general/kusama/kusama-parameters.md]:
```suggestion
A maximum of 24 validators can be nominated by a
```

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**